ui/lib/core/addon/templates/components/replication-action-demote.hbs

```diff
@@ -1,9 +1,9 @@
 <div class="action-block is-rounded" data-test-demote-replication>
   <div class="action-block-info">
-    <h4 class="title is-5 is-marginless">
```

Comment:
When replication was redone, we didn't have a consistent pattern for sub titles like this, so I had used h4. Keeping to the most popular pattern I changed to h3, and this had no visual change.